### PR TITLE
Removes Lintian warning and errors

### PR DESCRIPTION
**Pull request by fcastilloec**
The warnings `non-standard-dir-perm`, `non-standard-file-perm`, `extra-license-file` are addressed by this PR.
As we stand, only `debian-changelog-file-missing` and `binary-without-manpage` are the only lintianOverrides needed. Any other warning or errors are related to the app being packaged and not `electron-installer-debian`
The test provided uses `app-with-asar` so there's only one file permission to worried. All files and directories permissions inside `app-with-asar` have been change from `0664` -> `0644` and `0775` -> `0755`. This change is in accordance with debian rules and will pass all lintian checks address by this PR.

### CODE REVIEW

**Comment by fcastilloec** [comment]
I have rebased my PR with all the latest commits. @malept or @unindented could you briefly check the code and either approve or request changes before I merge this request?

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ addons:
     packages:
       - fakeroot
       - lintian
+      - fakeroot
```
Comment (unindented) [inline .travis.yml]: This is duplicate.

File: src/installer.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ var glob = require('glob')
 var path = require('path')
 var temp = require('temp').track()
 var wrap = require('word-wrap')
+var mkdirp = require('mkdirp')
```
Comment (unindented) [inline src/installer.js]: `fs-extra` already has this functionality (https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureDir.md), so no need to add more dependencies.

File: src/installer.js [hunk 3]
```diff
@@ -276,11 +277,12 @@ var createBinary = function (options, dir, callback) {
   var binDest = path.join(binDir, options.name)
   options.logger('Symlinking binary from ' + binSrc + ' to ' + binDest)
 
-  async.waterfall([
-    async.apply(fs.ensureDir, binDir),
-    async.apply(fs.symlink, binSrc, binDest, 'file')
-  ], function (err) {
-    callback(err && new Error('Error creating binary file: ' + (err.message || err)))
+  mkdirp(binDir, '0755', function (err, made) {
```
Comment (unindented) [inline src/installer.js]: Ah, but `fs-extra` doesn't have chmod options. Oh well, another dependency it is then.

File: .travis.yml (hunk 1 above)
Comment (fcastilloec) [reply .travis.yml]: Good catch, the result of rebasing and `eslint` doesn't check for this

File: src/installer.js (hunk 3 above)
Comment (fcastilloec) [reply src/installer.js]: Yes, we're going to need a new dependency until `fs-extra` decides to include chmod options to their commands

File: src/installer.js (hunk 2 above)
Comment (malept) [reply src/installer.js]: Appears to be unofficially supported: https://github.com/jprichardson/node-fs-extra/issues/60

Comment (fcastilloec) [reply src/installer.js]: I guess when it becomes officially supported we can switch to using only `fs-extra` in the meantime we should stick with `mkdirp`

**Pull request merged**